src/status_im2/contexts/chat/composer/view.cljs

```diff
@@ -158,10 +160,10 @@
                        :opacity                           opacity
                        :background-y                      background-y
                        :theme                             theme}
-        props         (utils/init-props)
-        state         (utils/init-state)]
+        props         (utils/init-props) ; Non-reactive state
```

Comment:
I appreciate these comments, I had to jump to the source of these functions a couple times to remember what we meant by `props` and `state` since both functions return some state.

Sometimes when I comment code I realize I can just rename the function for better clarity. Even if it's verbose, `init-props` and `init-state` are only called once, so I think it's fine.

So one suggestion would be to remove the comment and rename the functions:

- `init-props` -> `init-non-reactive-state`
- `init-state` -> `init-reactive-state`

Reply:
Good suggestion :)